experiment_scheduler/task_manager/task_manager_server.py

```diff
@@ -202,13 +202,19 @@ def get_task_log(self, request, context):
         The response is sent by streaming.
         """
         log_file_path = osp.join(request.log_file_path, f"{request.task_id}_log.txt")
+        self.logger.info(log_file_path)
         try:
             with open(log_file_path, mode="rb") as file:
+                is_read = -1
```

Comment:
Setting is_read with constant (ex. constant value IS_READ) seems more clear.

Reply:
'is_read' is not constant, but variable. So, IS_READ could not pass the pylint.